src/client/Microsoft.Identity.Client/Platforms/Features/WinFormsLegacyWebUi/WebBrowserInterfaces.cs

```diff
@@ -291,7 +291,9 @@ public interface IDocHostUIHandler
 
         [ComImport]
         [Guid("D30C1661-CDAF-11D0-8A3E-00C04FC9E26E")]
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
These warnings are from NetCore3.1 because `WinFormsLegacyWebUi` was added in Msal.Desktop.